### PR TITLE
CCM release notes for Jan 4

### PR DESCRIPTION
**Pull request by charanya-j**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

Tested the local build. Building successfully.

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ Harness Cloud Cost Management is updated regularly. Review the notes below for d
...
+### What's new
+This release does not include new features.
+
+
+### Early access
+* Standardize your currency across Harness CCM (CCM-9280)
+  This release introduces Currency Preference that enables you to view the entire CCM application in your preferred currency for different cloud providers. This feature is behind a feature flag CCM_CURRENCY_PREFERENCES.
+
+* API implementation for the Currency Preferences feature (CCM-9632)
+  You can now select the currency in which you want to view your entire CM application from different currencies across different cloud providers by using the Currency Preference APIs. Go to [Harness API Documentation](https://apidocs.harness.io/) for more information.
```
Comment (rashminandasahoo) [inline release-notes/cloud-cost-management.md:24]: You can now use the Currency Preference API to select the currency in which you want to view your entire CM application across different cloud providers.

File: release-notes/cloud-cost-management.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ Harness Cloud Cost Management is updated regularly. Review the notes below for d
...
 :::
+## January 4, 2023
+
+### What's new
+This release does not include new features.
+
+
+### Early access
+* Standardize your currency across Harness CCM (CCM-9280)
+  This release introduces Currency Preference that enables you to view the entire CCM application in your preferred currency for different cloud providers. This feature is behind a feature flag CCM_CURRENCY_PREFERENCES.
```
Comment (rashminandasahoo) [inline release-notes/cloud-cost-management.md:21]: You can now view the entire CCM application in your preferred currency for different cloud providers.

File: release-notes/cloud-cost-management.md (hunk 1 above)
Comment (rashminandasahoo) [reply release-notes/cloud-cost-management.md:24]: Do both the tickets correspond to a single enhancement?

**Review by rashminandasahoo — CHANGES_REQUESTED** [review]

File: release-notes/cloud-cost-management.md (hunk 2 above)
Comment (charanya-j) [reply release-notes/cloud-cost-management.md:21]: This one was reviewed by Deepak. "This release introduces" is something that he suggested for a new feature.

File: release-notes/cloud-cost-management.md (hunk 1 above)
Comment (charanya-j) [reply release-notes/cloud-cost-management.md:24]: Rephrased as suggested. Thank you.
Yes, users can use APIs also for this feature, hence mentioned separately.

**Review by rashminandasahoo — APPROVED** [review]

**Pull request merged**